src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -705,6 +705,68 @@ public static MetadataReference CreateMetadataReferenceFromIlSource(string ilSou
             return CompileAndVerify(comp);
         }
 
+        protected List<SyntaxNode> GetSyntaxNodeList(SyntaxTree syntaxTree)
```

Comment:
Sure, I can make that change.

---
In reply to: [123880121](https://github.com/dotnet/roslyn/pull/20364#discussion_r123880121) [](ancestors = 123880121)